extensions/interactions/PencilCodeEditor/PencilCodeEditor.js

```diff
@@ -33,8 +33,8 @@ oppia.directive('oppiaInteractivePencilCodeEditor', [
             pencilCodeEditorRulesService) {
           $scope.initialCode = oppiaHtmlEscaper.escapedJsonToObj(
             $attrs.initialCodeWithValue);
-
-          var pce = new PencilCodeEmbed($element[0].children[0]);
+          var iframeDiv = document.getElementById('pencil-code-editor-iframe');
```

Comment:
Mm, not so good -- this references the entire DOM, whereas the stuff that this extension does should really be encapsulated within the extension.

Can we instead give the div a class (not an id) and do something like $element[0].getSubElementByClassName? I.e. something like:

    http://stackoverflow.com/questions/6933763/how-do-i-select-item-with-class-within-a-div-in-jquery